client/app/views/register/my_accounts.coffee

```diff
@@ -3,6 +3,11 @@
 module.exports = class MyAccountsView extends LayoutView
```

Comment:
I understand, but I think `register` is not the correct naming for this folder (a folder to include all views). In prefer `views/onboarding` instead. But I think it's about preferences here.